src/components/business/AddIdentity.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import { HttpVerbs } from "../../@types/HttpVerbs";
+import { IdentityTypes } from "../../@types/Identity";
+import { useCookies } from "../../hooks/useCookies";
+import { fetchJson } from "../../utils/fetchJson";
+
+interface AddIdentityProps {
+  type: IdentityTypes;
+  children: (props: {
+    addIdentity: (value: string) => Promise<Response>;
+  }) => JSX.Element;
+}
+
+export const AddIdentity: React.FC<AddIdentityProps> = ({ type, children }) => {
+  const { data, error } = useCookies();
```

Comment:
If all our little component do that call, we may need to think about using context somewhere to avoid calling the API every time we instanciate a component (for now, I think that's acceptable but to have that in mind)

Reply:
I agree with that. I'll write a task to keep this in mind.

Reply:
For now, it works as an auth gate, but when we'll implement an auth middle, we'll be able to refactor this into a context that just provide the user ID.